CHANGES/7.misc

```diff
@@ -0,0 +1 @@
+Remove no longer needed execution environment logic and tests.
```

Comment:
this file should probably be a `.removal`

Reply:
didn't realize that was an option, thanks!